docs/aerodromes/classd/Bankstown.md

```diff
@@ -82,14 +88,14 @@ Due to the close proximity in critical stages of flight, there are additional co
 At Bankstown, all Runways are considered Independent unless a Multi-Engined OR Jet aircraft departs from an adjacent Runway. 
 In this instance, runways are treated as dependent *for departures ONLY*. Controllers should apply the same Runway standard across all dependant Runways. Arriving aircraft are exempt from this clause.
 
-Definitions;
-Independent: Runways can operate simultaneously without restriction.
-Dependent: Runways are operated with restrictions, often one at a time.
+**Independent**: Runways can operate simultaneously without restriction.  
+**Dependent**: Runways are operated with restrictions, often one at a time.
 
-Example A; A King Air (Multi-Engine) is ready to depart Runway 29R. A Cherokee is departing Runway 29L. Runways are not adjacent, and are therefore independent of each other.
-Example B; A Citation (Jet) is ready Runway 11C. Both Runways 11L & 11R are adjacent, and a Runway Separation Standard should be applied across ALL three Runways. Note: Arriving Aircraft are exempt.
-Example C; A PC12 (Single Engine) is ready Runway 29C. There aircraft is a single-engined aircraft and the Runways are treated independently from one another.
-Example D; A Seminole (Multi-Engine) is cleared a Touch & Go on Runway 11R. Any aircraft departing from the adjacent Runway (11C) must have a Runway separation standard applied.
+!!! example "Examples"
+    - A King Air (Multi-Engine) is ready to depart Runway 29R. A Cherokee is departing Runway 29L. Runways are not adjacent, and are therefore independent of each other.
+    - A Citation (Jet) is ready Runway 11C. Both Runways 11L & 11R are adjacent, and a Runway Separation Standard should be applied across ALL three Runways. Note: Arriving Aircraft are exempt.
+    - A PC12 (Single Engine) is ready Runway 29C. There aircraft is a single-engined aircraft and the Runways are treated independently from one another.
```

Comment:
Typo: "There" changed to "This"?

Or more succinct; "This is a single engined aircraft..."